flink-container/kubernetes/README.md

```diff
@@ -17,6 +17,7 @@ The files contain the following variables:
 
 - `${FLINK_IMAGE_NAME}`: Name of the image to use for the container
 - `${FLINK_JOB}`: Name of the Flink job to start (the user code jar must be included in the container image)
+- `${FLINK_JOB_ARGUMENS}`: Job specific command line arguments
```

Comment:
Shall we give a example or more documentation to guide how to pass the command line arguments? for example the format, like "--arg val" or something else? because here are many formats such as "--key value", "-Dxxx=xx".

Reply:
I removed it for the time being as in fact I haven't addresed the kubernetes part yet.